curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-algorithmic-thinking-by-building-a-dice-game/649a78fd5dd71202affcffcb.md

```diff
@@ -0,0 +1,275 @@
+---
+id: 649a78fd5dd71202affcffcb
+title: Step 3
+challengeType: 0
+dashedName: step-3
+---
+
+# --description--
+
+You need to create a function named `rollDice` that will be used to roll the dice and update the dice display on the result screen.
```

Comment:
```suggestion
Use the `function` keyword to declare a function named `rollDice`. This will be used to roll the dice and update the dice display on the result screen.
```